source/common/grpc/stat_names.h

```diff
@@ -21,6 +21,8 @@ struct StatNames {
   Stats::StatName streams_total_;
   std::array<Stats::StatName, Status::WellKnownGrpcStatus::MaximumKnown + 1> streams_closed_;
   absl::flat_hash_map<std::string, Stats::StatName> status_names_;
+  // Stat name tracking the creation of the google grpc client
```

Comment:
nit. Sorry, please ends the sentence with a period. Please check on other places as well. 🙏🏽

Reply:
my bad. Just searched all added comments and s/google/Google/g and added '.'